addons/mail/static/src/models/thread/thread.js

```diff
@@ -2169,6 +2177,12 @@ function factory(dependencies) {
         hasActivities: attr({
             default: false,
         }),
+        /**
+         * Determines whether the buttons to start a RTC call should be displayed.
+         */
+        hasCallButtons: attr({
+            compute: '_computeHasCallButtons',
+        }),
```

Comment:
This field should be defined on `ThreadViewTopbar` model.

Reply:
@alexkuhn we need to override `_computeHasCallButtons` in im_livechat but there is no thread_view_topbar.js file that already exists. So is it suitable to create a new file(thread_view_topbar.js) in a stable version?

Reply:
It's fine to create new file in stable since manifest are in python files (it was a problem in the past when it was xml).